### PR TITLE
change install messages to mention plguin manager, render report as html

### PR DESCRIPTION
**Pull request by psifertex**
Please note that this is /completely untested/ as i didn't have any coverage reports to play with. Just wanted to highlight that by using the HTML report you can make the addresses clickable which should improve the usability of the report, but feel free to let this serve as inspiration more than anything if you like. :-)

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -428,7 +430,11 @@ def show_coverage_report(bv):
         function_addr = function_dict[mangled_name].start
         report += "  0x%08x  %s%s : %.2f%% coverage\t( %-3d / %3d blocks)\n" % \
                   (function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
-    show_plain_text_report(title, report)
+        report_html += "<tr><td><a href='binaryninja:?expr=0x%x'>0x%08x</a></td><td>%s</td><td>%.2f%% coverage</td><td>%-3d / %3d blocks</td></tr>\n" % \
+                  (function_addr, function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
+
+    report_html += "<table>\n"
```
Comment (r0ck3tAKATrashPanda) [inline __init__.py]:
```suggestion
    report_html += "</table>\n"
```

File: __init__.py [hunk 2]
```diff
@@ -428,7 +430,11 @@ def show_coverage_report(bv):
...
+    show_html_text_report(title, report_html, plaintext=report)
```
Comment (r0ck3tAKATrashPanda) [inline __init__.py]:
```suggestion
    show_html_report(title, report_html, plaintext=report)
```

File: __init__.py [hunk 3]
```diff
@@ -428,7 +430,11 @@ def show_coverage_report(bv):
         function_addr = function_dict[mangled_name].start
         report += "  0x%08x  %s%s : %.2f%% coverage\t( %-3d / %3d blocks)\n" % \
                   (function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
-    show_plain_text_report(title, report)
+        report_html += "<tr><td><a href='binaryninja:?expr=0x%x'>0x%08x</a></td><td>%s</td><td>%.2f%% coverage</td><td>%-3d / %3d blocks</td></tr>\n" % \
+                  (function_addr, function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
```
Comment (r0ck3tAKATrashPanda) [inline __init__.py]:
```suggestion
                  (function_addr, function_addr, name, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
```

You would either need to make `%s` into `%s%s` in the previous line, or remove this. The table should format for you regardless.

File: __init__.py [hunk 4]
```diff
@@ -428,7 +430,11 @@ def show_coverage_report(bv):
         function_addr = function_dict[mangled_name].start
         report += "  0x%08x  %s%s : %.2f%% coverage\t( %-3d / %3d blocks)\n" % \
                   (function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
-    show_plain_text_report(title, report)
+        report_html += "<tr><td><a href='binaryninja:?expr=0x%x'>0x%08x</a></td><td>%s</td><td>%.2f%% coverage</td><td>%-3d / %3d blocks</td></tr>\n" % \
```
Comment (r0ck3tAKATrashPanda) [inline __init__.py]:
```suggestion
        report_html += "<tr><td><a href='binaryninja://?expr=0x%x'>0x%08x</a></td><td>%s</td><td>%.2f%% coverage</td><td>%-3d / %3d blocks</td></tr>\n" % \
```

Double slashes are needed according to API documentation for this.

**Comment by r0ck3tAKATrashPanda** [comment]
Clickable links++ Tested with ascii drcov format file and it worked like a charm with a few very minor changes.

**Comment by mark-griffin** [comment]
@r0ck3tAKATrashPanda thanks for the updates, looks like it works.  I can't commit the suggestions, I think @psifertex has to, but if he doesn't get to it in the next day or so I'll just integrate them myself and mention you.

File: __init__.py (hunk 3 above)
Comment (psifertex) [reply __init__.py]: Thanks, this was my intention. As I said, I didn't actually test it. 😉

**Comment by psifertex** [comment]
Thanks testing it, I figured there would be bugs as i couldn't actually try it. :-)

**Pull request merged**